test/e2e/network/endpointslice.go

```diff
@@ -562,3 +565,17 @@ func createServiceReportErr(cs clientset.Interface, ns string, service *v1.Servi
 	framework.ExpectNoError(err, "error deleting Service")
 	return svc
 }
+
+func logEndpointSlice(epSlice discoveryv1beta1.EndpointSlice) {
+	ports := []string{}
+	for _, port := range epSlice.Ports {
+		ports = append(ports, fmt.Sprint(*port.Port))
```

Comment:
this can be null, don't dereference without checking?

Reply:
Updated, thanks!